styles.css

```diff
@@ -0,0 +1,50 @@
+.all {
+  height: 916px;
+  top: 1px;
+  border-radius: 1px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  height: 72px;
+  width: 100vw;
+}
+
+.main {
+  margin-top: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-image: url("./images/Header bg@2x.png");
+  background-size: 100% 100%;
+}
+
+h1 {
+  width: 400px;
+  font-weight: 700;
+  margin-top: 100px;
+}
+
+p {
+  width: 400px;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
- [ ] Great job with your project 👏. As the previous reviewers mentioned, kindly ensure your design is responsive and can be seen. Kindly fix this 🙏 . One of the reasons your design isn't clear and responsive is because of the `width` given to almost all elements.

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205286014-9a0345c2-7fe4-4bbf-bb9e-1ae157d0f609.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205285029-87c54dda-182b-4331-85a5-45cf60df9a63.png)